middlewares/hacker.middleware.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+const TAG = `[ HACKER.MIDDLEWARE.js ]`;
+const mongoose = require("mongoose");
+
+function parseHacker(req, res, next) {
+    const hackerDetails = {
+        _id: mongoose.Types.ObjectId(),
+        accountId: req.body.accountId,
+        status: "Applied",
```

Comment:
Should status: "Applied" always be the case here? I feel like this might be the start of a nasty bug that we can't catch

Reply:
Put it in another function as discussed.